apps/client/components/ui/Dashboard/NominationPhase.tsx

```diff
@@ -0,0 +1,27 @@
+import { Panel, PanelContent, PanelHeader } from '../Panel'
```

Comment:
yo, just a heads up that I'm currently working on the NominationPanelContent.

this doesn't conflict so far, but I want to make sure you're aware :)

Reply:
ok ill remove